### PR TITLE
Add `speaker_tasks` datasets folder, add diarization datasets voxconverse/aishell

### PR DESCRIPTION
**Pull request by SeanNaren**
# What does this PR do ?

Adds a folder to store the `speaker_tasks` datasets. Few things to consider:

* I could create a nested `diarization` folder to be more specific, but I thought just one folder would be fine
* I might've missed some scripts that would be better suited to exist in this new folder

# Changelog 
- Add `speaker_tasks` datasets folder, add diarization datasets voxconverse/aishell.

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [x] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

### CODE REVIEW

File: scripts/dataset_processing/speaker_tasks/get_voxconverse.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        f.write('\n'.join([str(os.path.join(rttm_path, x)) for x in os.listdir(rttm_path)]))
+    create_manifest(
+        audio_list, str(manifest_output_path), rttm_path=rttm_list,
+    )
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Aishell Data download")
+    parser.add_argument("--data_root", default='./', type=str)
+    parser.add_argument("--output_manifest_path", default='aishell_diar_manifest.json', type=str)
```
Comment (nithinraok) [inline scripts/dataset_processing/speaker_tasks/get_voxconverse.py:55]: update aishell to voxconverse name here

**Review by nithinraok — COMMENTED** [review]
Sean, can you also move hi-mia data to speaker_tasks

**Comment by SeanNaren** [comment]
@nithinraok could i get a re-review?

**Comment by nithinraok** [comment]
`tutorials/speaker_tasks/Speaker_Identification_Verification.ipynb` This notebook needs to be updated to use latest path of hi-mia. rest looks good to me.

**Comment by SeanNaren** [comment]
done @nithinraok!

**Comment by nithinraok** [comment]
Thanks Sean. LGTM

**Pull request merged**